docs/tutorial.md

```diff
@@ -104,12 +88,10 @@ hello-prisma2
     └── schema.prisma
 ```
 
-> **Note**: If you were using **SQLite**, make sure that you deleted all files that were generated initially and only keep the `prisma/schema.prisma` file!
-
 `schema.prisma` is your [Prisma schema file](./prisma-schema-file.md). It generally contains three important elements for your project:
 
 - Data sources (here, that's your PostgreSQL database)
-- Generators (you'll add this soon)
+- Generators (here, that's the generator for Photon.js)
```

Comment:
Is this the right terminology here? I am still unsure if it's "Photon.js" or "PhotonJS".

Reply:
Yes, this has been resolved. We're referring to it as `Photon.js` (you can refer to [this](https://prisma-company.slack.com/archives/CEYCG2MCN/p1567081308007800?thread_ts=1567081184.006000&cid=CEYCG2MCN) internal Slack discussion for context).